compiler/rustc_lexer/src/lib.rs

```diff
@@ -225,14 +225,15 @@ pub fn first_token(input: &str) -> Token {
 }
 
 /// Creates an iterator that produces tokens from the input string.
-pub fn tokenize(mut input: &str) -> impl Iterator<Item = Token> + '_ {
+#[rustfmt::skip]
```

Comment:
Why are we skipping rustfmt here? That seems unexpected.

Reply:
Otherwise it formats the `from_fn` call to a single line, which I find unreadable.